instances/devhub.near/widget/devhub/components/molecule/SimpleMDE.jsx

```diff
@@ -139,7 +140,13 @@ function getSuggestedAccounts(term) {
   let results = [];
 
   term = (term || "").replace(/\W/g, "").toLowerCase();
-  const limit = 5;
+  let limit = 5;
+  if (term.length < 2) {
```

Comment:
why do we have this term length check?

Reply:
If a user doesn't type anything, the most relevant users will appear first. However, if a user types a specific name, the list should display the matching name instead of keeping the most relevant users at the top, followed by the match.